ADVANCED.md

```diff
@@ -122,20 +122,18 @@ Please see the notes below for link resolution prior to v.10.0.0 and v.7.0.0.
 
 #### Note: link resolution for versions older than 10.0.0
 
-Please note that for versions older than 10.0.0, disabling link resolution needs to be done via [configuration options](README.md#response-configuration-options) during client creation.
-To disable it, set `resolveLinks` to `false` when creating the Contentful client. Like so:
+Please note that for versions older than 10.0.0, disabling link resolution needs to be done via [Chained Clients](README.md#chained-clients).
```

Comment:
No, chained clients are for v10. This didn’t change.

Reply:
oh, nice catch